minikube-ingress-dns-macos

```diff
@@ -15,15 +15,16 @@ if [[ "$1" == "start" ]]; then
   ip="$(minikube ip)"
   echo "The IP address of running a cluster: ${ip}"
 
-  config="address=/${MINIKUBE_INGRESS_DNS_DOMAIN}/${ip}"
+  config="address=/${MINIKUBE_INGRESS_DNS_DOMAIN}/${ip}
+except-interface=${ip}""
```

Comment:
According to the man page of dnsmasq, `--except-interface` seems to be able to take an interface name only. Does it work correctly?
> -I, --except-interface=\<interface name\>